Assignment2/Database.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Establish database connection and perform insertion.
+ */
+class Database {
+
+  // Instance variables to store credentials.
+  private $server_name;
+  private $user_name;
+  private $password;
+  private $db_name;
+
+  // Connection object.
+  public $conn;
+
+  // Variables for holding queries.
+  public $q1;
```

Comment:
Please update the variable names.
The name of the variable must convey some information about the kind of data it should store or their purpose.
Simply writing q1,q2 is not acceptable